tools/testing/selftests/bpf/bpf_helpers.h

```diff
@@ -144,6 +144,39 @@ static unsigned long long (*bpf_skb_cgroup_id)(void *ctx) =
 static unsigned long long (*bpf_skb_ancestor_cgroup_id)(void *ctx, int level) =
 	(void *) BPF_FUNC_skb_ancestor_cgroup_id;
 
+/* #ifdef CONFIG_XDP_LUA */
+static int (*bpf_lua_dataref)(void *ctx, int offset) =
+    (void *)BPF_FUNC_lua_dataref;
+static void (*bpf_lua_dataunref)(void *ctx, int data_ref) =
+    (void *)BPF_FUNC_lua_dataunref;
+static int (*bpf_lua_pcall)(const char *funcname, int num_args, int num_rets) =
+    (void *) BPF_FUNC_lua_pcall;
+static void (*bpf_lua_pop)(int index) =
+    (void *)BPF_FUNC_lua_pop;
+static void (*bpf_lua_pushinteger)(int num) =
+    (void *)BPF_FUNC_lua_pushinteger;
+static void (*bpf_lua_pushlightuserdata)(void *ptr) =
+    (void *)BPF_FUNC_lua_pushlightuserdata;
+static void (*bpf_lua_pushlstring)(const char * str, size_t len) =
+    (void *)BPF_FUNC_lua_pushlstring;
+static void (*bpf_lua_pushmap)(void *map) =
+    (void *)BPF_FUNC_lua_pushmap;
+static void (*bpf_lua_pushskb)(void) =
+    (void *)BPF_FUNC_lua_pushskb;
+static void (*bpf_lua_pushstring)(const char *str) =
+    (void *)BPF_FUNC_lua_pushstring;
+static int (*bpf_lua_toboolean)(int index) =
+    (void *)BPF_FUNC_lua_toboolean;
+static int (*bpf_lua_tointeger)(int index) =
+    (void *)BPF_FUNC_lua_tointeger;
+static int (*bpf_lua_newpacket)(void *ctx, int offset) =
+    (void *)BPF_FUNC_lua_newpacket;
+static int (*bpf_lua_tostring)(char *sslsni, u32 size, int index) =
```

Comment:
keep consistent with the rest of the file. (Also the definition present in `linux/types.h`)

```suggestion
static int (*bpf_lua_tostring)(char *sslsni, __u32 size, int index) =
```